src/Raven.Studio/wwwroot/App/views/resources/addNewShardToDatabaseGroup.html

```diff
@@ -1,26 +1,18 @@
 <div class="modal-dialog add-new-node-to-db-group" role="document">
-    <div class="modal-content" tabindex="-1">
+    <div class="modal-content neo-modal" tabindex="-1">
```

Comment:
did we already defined neo-modal somewhere?

Reply:
It is used in the `shardSelector` view